docs/src/components/Hero/SelectedLabel/SelectionBox/index.tsx

```diff
@@ -53,18 +53,42 @@ const SelectionBox: React.FC<{
   children,
   isInteractive,
 }) => {
+  const touchPosition = { x: 0, y: 0 };
+
   const classList = getClassListByIdentifier(
     draggableIdentifier,
     isInteractive
   );
 
   return (
     <Draggable
+      onStart={(event: any) => {
+        if (
+          typeof event.movementX === 'number' &&
+          typeof event.movementY === 'number'
+        )
+          return;
```

Comment:
Why `typeof event.movementX === 'number'` is a problem?

Reply:
The problem is that `Reanimated` text handled mouse events only. So it way to check whether event is touch or click.